tomb/src/main.rs

```diff
@@ -3,6 +3,7 @@
 #![feature(buf_read_has_data_left)]
 #![cfg_attr(docsrs, feature(doc_cfg))]
 #![warn(missing_debug_implementations, missing_docs, rust_2018_idioms)]
+#![deny(private_interfaces)]
```

Comment:
Not familiar with this one. Looked it up and seems good to me

Reply:
yeah, one of the old lints we had became deprecated and this is its successor